### PR TITLE
Update to backstage release 1.0 plugins

### PR DESCRIPTION
**Pull request by jrusso1020**
## Change description
Update to backstage release 1.0 and migrate to package roles
> Description here

## Type of change

- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [x] Improvement (improves codebase without changing functionality)

## Related issues

> Fix [#1]()

### Development

- [x] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [x] Automated tests covering modified code pass

### Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Comment by jrusso1020** [comment]
@nikhilunni someone will need to validate this, there was a number of breaking changes I had to go in and attempt to fix

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "0.1.35",
+  "version": "0.1.36",
```
Comment (jrusso1020) [inline package.json:3]: should this be a minor update?

Comment (cristinabuenahora) [reply package.json:3]: yeah but I just merged something in so you might want to bump again :)

Comment (cristinabuenahora) [reply package.json:3]: yeah but I just merged something in so you might want to bump again :)

**Review by nikhilunni — DISMISSED** [review]

**Review by nikhilunni — DISMISSED** [review]

**Review by nikhilunni — APPROVED** [review]

**Pull request merged**